eos/rare-b-decays/lambdab-to-lambdac-l-nu.cc

```diff
@@ -0,0 +1,383 @@
+            inline double k10() const { return _k[9]; }
+
+            inline double decay_width() const
+            {
+                return 2.0 * k1ss() + k1cc();
+            }
+
+            inline double a_fb_leptonic() const
+            {
+                return 3.0 / 2.0 * k1c() / decay_width();
+            }
+
+            inline double a_fb_hadronic() const
+            {
+                return 1.0 / 2.0 * (2.0 * k2ss() + k2cc()) / decay_width();
+            }
+
+            inline double a_fb_combined() const
+            {
+                return 3.0 / 4.0 * k2c() / decay_width();
+            }
+
+            inline double f_zero() const
+            {
+                return (2.0 * k1ss() - k1cc()) / decay_width();
+            }
+        };
+    }
+
+    /**/
+
+    template <> struct Implementation<LambdaBToLambdaCLeptonNeutrino>
+    {
+        std::shared_ptr<Model> model;
+
+        Parameters parameters;
+
+        UsedParameter hbar;
+        UsedParameter tau_Lambda_b;
+
+        UsedParameter g_fermi;
+
+        SwitchOption opt_l;
+        UsedParameter m_l;
+
+        UsedParameter m_Lambda_b;
+        UsedParameter m_Lambda_c;
+        UsedParameter alpha;
+
+        UsedParameter mu;
+
+        std::shared_ptr<FormFactors<OneHalfPlusToOneHalfPlus>> form_factors;
+
+        Implementation(const Parameters & p, const Options & o, ParameterUser & u) :
+            model(Model::make(o.get("model", "SM"), p, o)),
+            parameters(p),
+            hbar(p["hbar"], u),
+            tau_Lambda_b(p["life_time::Lambda_b"], u),
+            g_fermi(p["G_Fermi"], u),
+            opt_l(o, "l", {"e", "mu", "tau"}, "mu"),
+            m_l(p["mass::" + opt_l.value()], u),
+            m_Lambda_b(p["mass::Lambda_b"], u),
+            m_Lambda_c(p["mass::Lambda_c"], u),
+            alpha(p["Lambda_c::alpha@FOCUS"], u),
+            mu(p["mu"], u)
+        {
+            form_factors = FormFactorFactory<OneHalfPlusToOneHalfPlus>::create("Lambda_b->Lambda_c::" + o.get("form-factors", "DKMR2017"), p, o);
+
+            if (! form_factors.get())
+                throw InternalError("Form factors not found!");
+
+            u.uses(*form_factors);
+            u.uses(*model);
+
+        }
+
+        const complex<double> norm(const double & s) const
+        {
+            return g_fermi() * 4.0 * M_PI * model->ckm_cb() * std::sqrt(s / 3.0 / 2048 / std::pow(M_PI, 5.0) /power_of<3>(m_Lambda_b()) * std::sqrt(lambda(m_Lambda_b * m_Lambda_b, m_Lambda_c * m_Lambda_c, s)));
+        }
+
+        lambdab_to_lambdac_l_nu::Amplitudes amplitudes(const double & s)
+        {
+            lambdab_to_lambdac_l_nu::Amplitudes result;
+
+            // define below the b->c WCs and the translation rule from BFvD2014 to EOS basis
+            const WilsonCoefficients<BToC> wc = model->wilson_coefficients_b_to_c(opt_l.value(), false);
+
+            const complex<double> c9        = wc.cvl() / 2.0;
```

Comment:
Ok, this I don't like. It's the easy way out. Please remove any mention of c9, c10 and so on from this file and replace them with cVL, cVR and so on.